build/CoreGraphics/dll/CoreGraphics.vcxproj

```diff
@@ -51,7 +51,7 @@
     </ClCompile>
     <Link>
       <ModuleDefinitionFile>CoreGraphics.def</ModuleDefinitionFile>
-      <AdditionalDependencies>cairo.lib;pixman.lib;freetype.lib;libpng.lib;libz.lib;libjpeg.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>dxguid.lib;libdispatch.lib;cairo.lib;pixman.lib;freetype.lib;libpng.lib;libz.lib;libjpeg.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
I think there are still depending components here; they'll be dropped the moment it becomes feasible.

---
In reply to: [82915584](https://github.com/Microsoft/WinObjC/pull/1126#discussion_r82915584) [](ancestors = 82915584)